core/server/config/overrides.json

```diff
@@ -64,5 +64,12 @@
     },
     "maintenance": {
         "enabled": false
+    },
+    "api": {
+        "versions": {
+            "active": "v2",
```

Comment:
How does it look when i ask for the active admin url?

Reply:
Updated the structure to handle admin/client URLs